genesyscloud/architect_user_prompt/genesyscloud_architect_user_prompt_proxy.go

```diff
@@ -155,11 +156,20 @@ func deleteArchitectUserPromptFn(_ context.Context, p *architectUserPromptProxy,
 	return nil, nil
 }
 
-func getAllArchitectUserPromptsFn(_ context.Context, p *architectUserPromptProxy, includeMediaUris, includeResources bool, name string) (*[]platformclientv2.Prompt, *platformclientv2.APIResponse, error) {
+func getAllArchitectUserPromptsFn(_ context.Context, p *architectUserPromptProxy, includeMediaUris, includeResources bool, name string, exportNames []string) (*[]platformclientv2.Prompt, *platformclientv2.APIResponse, error) {
 	const pageSize = 100
 	var allPrompts []platformclientv2.Prompt
+	var nameFilter []string
 
-	userPrompts, response, err := p.architectApi.GetArchitectPrompts(1, pageSize, []string{name}, "", "", "", "", includeMediaUris, includeResources, nil)
+	// Determine whether to pass in a single name for getByName or export name filter
```

Comment:
Why not leave it as is and just pass in to the `name` argument the first character of each letter in the `getAllUserPrompts()`?